packages/foreman/foreman/foreman.spec

```diff
@@ -198,19 +153,9 @@ BuildRequires: (npm(os-browserify) >= 0.3.0 with npm(os-browserify) < 1.0.0)
 BuildRequires: (npm(react-intl) >= 2.8.0 with npm(react-intl) < 3.0.0)
 # end package.json dependencies BuildRequires
 
-# start specfile assets BuildRequires
-BuildRequires: (rubygem(patternfly-sass) >= 3.59.4 with rubygem(patternfly-sass) < 3.60.0)
-BuildRequires: (rubygem(gettext_i18n_rails_js) >= 1.4 with rubygem(gettext_i18n_rails_js) < 2.0)
-BuildRequires: (rubygem(po_to_json) >= 1.1 with rubygem(po_to_json) < 2.0)
-BuildRequires: (rubygem(execjs) >= 1.4.0 with rubygem(execjs) < 3.0)
-BuildRequires: (rubygem(terser) >= 1.1 with rubygem(terser) < 2.0)
-BuildRequires: (rubygem(sass-rails) >= 6.0 with rubygem(sass-rails) < 7.0)
-# end specfile assets BuildRequires
-
-# start specfile facter BuildRequires
-BuildRequires: rubygem(facter)
-# end specfile facter BuildRequires
-
+%generate_buildrequires
```

Comment:
This can also be used on normal gems, right?
The script is only needed because core is not a gem?

Reply:
There is no generic implementation for gems because there's no good way to determine what's needed. But we can implement this ourselves.

I'd like to have something for Foreman plugins as well but thought to first focus on Foreman itself since I think it's where we get the most benefits.

Also converting nodejs is a good next step but I thought it was already complex enough now